app/src/pages/inside/stepPage/stepGrid/stepGrid.jsx

```diff
@@ -306,13 +320,17 @@ export class StepGrid extends Component {
           },
           onUnlinkSingleTicket,
           events: {
-            onEditEvent: events.MAKE_DECISION_MODAL_EVENTS.openModal,
+            onEditEvent:
+              events.MAKE_DECISION_MODAL_EVENTS && events.MAKE_DECISION_MODAL_EVENTS.openModal,
           },
         },
         withFilter: true,
         filterEventInfo: events.DEFECT_TYPE_FILTER,
       },
     ];
+    if (Object.keys(modifyColumnsSettings).length) {
+      this.columns = modifyColumnsFunc(this.columns, modifyColumnsSettings);
```

Comment:
I think that `modifyColumnsSettings` prop is redundant here. You may use `modifyColumnsFunc` and it will update columns according to its implementation on the top level.